delfin/drivers/huawei/oceanstor/rest_client.py

```diff
@@ -244,4 +244,11 @@ def get_all_pools(self):
 
     def clear_alert(self, sequence_number):
         url = "/alarm/currentalarm?sequence=%s" % sequence_number
-        return self.call(url, method="DELETE", log_filter_flag=True)
+
+        # Result always contains error code and description
+        result = self.call(url, method="DELETE", log_filter_flag=True)
+        if result['error']['code'] != 0:
+            msg = (_('Clear alert failed with reason: %s.')
```

Comment:
InvalidResults(msg), already cover internation language format. so you may remove _ (for ineternation lang.)

msg = ('Clear alert failed with reason: %s.')% result['error']['description'])

Reply:
Updated